ush/python/pygfs/task/marine_letkf.py

```diff
@@ -205,5 +205,9 @@ def finalize(self):
         # stage the desired diag files
         FileHandler({'mkdir': [os.path.join(letkfsaveconf.COMOUT_OCEAN_LETKF, 'diags')]}).sync()
         FileHandler({'copy': obs_files_to_copy}).sync()
+        # yaml configurations
+        yamls_to_copy = []
+        yamls_to_copy.append([letkfsaveconf.letkf_yaml_file, os.path.join(letkfsaveconf.COMOUT_OCEAN_LETKF, 'letkf.yaml')])
```

Comment:
This action is fine.   
However, we are saving all the configurations to directory named `COM_CONF` instead of `COMOUT_OCEAN_LETKF`.  `COM_CONF` is a directory in `COM` that has all the configurations of all the executables that are run in a cycle.

Reply:
Nice, I didn't know this! I'll take a look later this week to change this and maybe also some other marine yamls that get copied to a different directory.

Reply:
Finally getting back to this. Is there a place where things from COM_CONF get archived? I haven't found it in the code yet. I'm wondering if I need a custom addition for archiving this yaml, or if it's already handled some place.